circleci.sh

```diff
@@ -89,7 +89,7 @@ coverage() {
 
     # rebuild dmd with coverage enabled
     # use the just build dmd as host compiler this time
-    mv src/dmd src/host_dmd
+    mv generated/dmd src/host_dmd
```

Comment:
So, if this PR goes further all the scripts in the dlang repo must be updated. Is this hard to accomplish?